### PR TITLE
[WIP] Add project initialization / `builder init`

### PR DESCRIPTION
**Pull request by ryan-roemer**
For #7 

Please let @chaseadamsio know if you are continuing this work, branch this branch and work on that.

**DO NOT MERGE**

### CODE REVIEW

**Comment by ryan-roemer** [comment]
Hey @chaseadamsio -- I'm going to be diving into this hopefully this week and taking this over. Thanks for the work so far and ping me if you want to dive back in! ;)

**Comment by chaseadamsio** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "builder",
-  "version": "2.1.1",
+  "version": "2.1.3",
```
Comment (coopy) [inline package.json]: I'd call this a minor bump (feature). So `2.2.0`.

File: lib/init.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+ * Installation of builder archetype and archetype-dev dependencies
+ *
+ * @returns {void}
+ */
+var installDeps = function (callback) {
+  log.info("init", "Installing dependency for " + archetype.name);
+
+  //var initInstallCmd = "npm install " + archetype.name + " --save" +
+    //" && npm install " + archetype.name + "-dev --save-dev";
+  var initInstallCmd = "echo 'hello'";
```
Comment (coopy) [inline lib/init.js]: Leftover?

File: lib/init.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+  runner.run(initInstallCmd, {}, function () {
+    try {
+      archetype.meta = require(path.join(process.cwd(), "node_modules", archetype.name, "meta.js"));
+    } catch (e) {
+      log.error(e);
```
Comment (coopy) [inline lib/init.js]: We probably only need one `log.error`.

Comment (coopy) [inline lib/init.js]: Should we `callback(e)`?

**Comment by ryan-roemer** [comment]
hehe @chaseadamsio , I'm shutting this down.

Thanks for all the POC work an inspiration!

**Comment by chaseadamsio** [comment]
:+1:

**Pull request closed without merging**